hyph-utf8/tex/generic/hyph-utf8/patterns/ptex/hyph-la-x-classic.ec.tex

```diff
@@ -0,0 +1,12028 @@
+% title: Hyphenation patterns for classical Latin
+% copyright: Copyright (C) 2019 Keno Wehr
+% notice: This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org/tex for more information.
+%     These patterns are also suitable for medieval and modern Latin if German
+%     or Slavic pronunciation is used.
+%     See https://github.com/gregorio-project/hyphen-la/tree/master/doc for
+%     documentation.
+% source: https://github.com/gregorio-project/hyphen-la/tree/master/patterns/generation
+% language:
+%     name: Classical Latin
+%     tag: la-x-classic
+% version: 2.0-EC 2019-07-03
+% authors:
+%  -
+%     name: Claudio Beccari
+%  -
+%     name: Keno Wehr
+%     contact: wehr (at) abgol.de
+% licence:
+%     name: MIT
+%     url:  http://opensource.org/licenses/MIT
+%     text: >
+%           Permission is hereby granted, free of charge, to any person
+%           obtaining a copy of this software and associated documentation
+%           files (the "Software"), to deal in the Software without
+%           restriction, including without limitation the rights to use,
+%           copy, modify, merge, publish, distribute, sublicense, and/or
+%           sell copies of the Software, and to permit persons to whom the
+%           Software is furnished to do so, subject to the following
+%           conditions:
+%
+%           The above copyright notice and this permission notice shall be
+%           included in all copies or substantial portions of the Software.
+%
+%           THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+%           EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+%           OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+%           NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+%           HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+%           WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+%           FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+%           OTHER DEALINGS IN THE SOFTWARE.
+% hyphenmins:
+%     generation:
+%         left: 2
+%         right: 2
+%     typesetting:
+%         left: 2
+%         right: 2
+% texlive:
+%     encoding: ec
+%     babelname: classiclatin
+%     message: Classical Latin hyphenation patterns
+%     package: latin
+% changes:
+%  -
+%     date: 2014-10-06
+%     version: 1.2
+%     author: Claudio Beccari
+%  -
+%     date: 2019-07-03
+%     version: 2.0-EC
+%     author: Keno Wehr
+%     description: patgen-based pattern generation; improved support of
+%         compound words; do not hyphenate Roman numerals; support of v and j
+%         spellings, ae/oe ligatures, and accents
+%
+\message{Hyphenation patterns for classical Latin, v. 2.0-EC 2019-07-03}
```

Comment:
Can you please add the string
    % ==========================================
(percent, space, 42 equals signs) over this line?  It signals the end of the YAML header; without it the extraction scripts fail.

It’s also partly redundant with the message line from `loadhyph-la-x-classic.tex`.
This line is partly redundant

Reply:
Which file do you prefer for the message? `loadhyph-la-x-classic.tex` or the pattern file `hyph-la-x-classic.tex` itself? I can of course remove one of the messages.
But note that German also has messages in both files.

Reply:
`loadhyph`, please.  I know that German has multiple messages too, and it’s not a good idea to have redundancy there either :-)

(When I get the time I’ll work with Werner to make the format of the German header file more consistent with other languages.)

Reply:
OK, done.
Please have a look.